### ISSUE TITLE
Label is not provided for the 'Heading size picker' combo box in windows.

### ISSUE THREAD

**Issue by msftedad** [quilljs/quill#4797]
**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26200.8039)
Browser: Edge (Version 147.0.3912.60), Chrome (Version 147.0.7727.56)
Screen Reader: NVDA (2025.3.3), JAWS (2026.2603.86), Narrator
URL: https://quilljs.com/playground/react

**User Impact:**
Screen reader users will be impacted as the ‘Heading size picker’ combo box does not have a label, making it difficult for them to understand its purpose while navigating through the interface.

**Pre-Requisite:** 
Turn on Screen Readers:
NVDA: Ctrl + Alt + N 
JAWS: Ctrl + Alt + J 
Narrator: Ctrl + win + Enter 
Verbosity:
NVDA: Default 
JAWS: Beginner, Highest 
Narrator: Default

**Repro Steps:**

1. Open the URL: https://quilljs.com/playground/react in edge/chrome browser.
2. Playground page will open.
3. Press tab key to navigate to the 'Heading size picker' control.
4. Verify whether label is provided for the 'Heading size picker' combo box in windows or not.

**Actual Result:**
Label is not provided for the 'Heading size picker' combo box in windows.
Screen reader narration: Normal button collapsed.
Only selected value appears within 'Heading size picker' combo box not visual label provided.

**Refer Attachment:**

1. Label is not provided for the 'Heading size picker' combo box in windows.png
2. Label is not provided for the 'Heading size picker' combo box in windows.mp4

https://github.com/user-attachments/assets/cb480301-0fe5-4086-a7f1-271b8823c12a
<img width="1920" height="1286" alt="Image" src="https://github.com/user-attachments/assets/dc519599-7d6d-413e-9360-c9f300636184" />

**Expected Result:**
Label as 'Heading size' should be provided for the 'Heading size picker' combo box in windows.
Screen Reader should announce as 'Heading size combo box normal collapsed'
